devices/devs.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Phoenix-RTOS
+ *
+ * phoenix-rtos-loader
+ *
+ * Devices Interface
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Hubert Buczynski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+
+#ifndef _DEVS_H_
+#define _DEVS_H_
+
+#include "../types.h"
+#include "config.h"
+
+#define DEV_UART_ID    0
+#define DEV_USB_ID     1
+#define DEV_FLASH_ID   2
+#define DEV_ID3        3    /* TBD */
+#define DEV_ID4        4    /* TBD */
+#define DEV_ID5        5    /* TBD */
+#define DEV_ID6        6    /* TBD */
+#define DEV_ID7        7    /* TBD */
+
+
+typedef struct {
+	int (*deinit)(u16 dn);
+
+	ssize_t (*read)(u16 dn, addr_t offs, u8 *buff, u32 len);
+	ssize_t (*write)(u16 dn, addr_t offs, const u8 *buff, u32 len);
+} dev_handler_t;
+
+
+extern void devs_regDriver(int dev, int (*init)(int dn, dev_handler_t *h));
```

Comment:
I would add comment that this function can be called only before `devs_init`, preferably from device drivers constructors

Reply:
Ok, it can be helpful.